schema/CompoundSchemaEnum.mcf

```diff
@@ -2504,3 +2504,88 @@ name: "DarpinAntibody "
 description: "DARPins (an acronym for designed ankyrin repeat proteins) are genetically engineered antibody mimetic proteins typically exhibiting highly specific and high-affinity target protein binding."
 descriptionUrl: "https://en.wikipedia.org/wiki/DARPin"
 
+Node: dcid:RelationshipAssociationTypeEnum
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+name: "RelationshipAssociationTypeEnum"
+description: "Type of relationship association as defined by PharmGKB, either 'associated', 'ambiguous', or 'not associated'."
+descriptionURL: "https://www.pharmgkb.org/page/downloadRelationshipsHelp"
+
+Node: dcid:RelationshipAssociationTypeAssociated
+typeOf: dcid:RelationshipAssociationTypeEnum
+name: "Associated"
+description: "Associated means an association between the entities is supported by the Evidence and PMIDs columns."
+descriptionURL: "https://www.pharmgkb.org/page/downloadRelationshipsHelp"
+
+Node: dcid:RelationshipAssociationTypeNotAssociated
+typeOf: dcid:RelationshipAssociationTypeEnum
+name: "NotAssociated"
+description: "Not associated means that the entities were evaluated but not found have a statistically significant association based on the PMIDs column."
+descriptionURL: "https://www.pharmgkb.org/page/downloadRelationshipsHelp"
+
+Node: dcid:RelationshipAssociationTypeAmbiguous
+typeOf: dcid:RelationshipAssociationTypeEnum
+name: "Ambiguous"
+description: "Ambiguous means that some of the items in the Evidence and/or PMIDs columns support an association and others do not."
+descriptionURL: "https://www.pharmgkb.org/page/downloadRelationshipsHelp"
+
+Node: dcid:RelationshipEvidenceTypeEnum
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+name: "RelationshipEvidenceTypeEnum"
+description: "Type of evidence for a relationship as defined by PharmGKB."
+descriptionURL: "https://www.pharmgkb.org/page/downloadRelationshipsHelp"
+
+Node: dcid:RelationshipEvidenceTypeClinicalAnnotation
+typeOf: dcid:RelationshipEvidenceTypeEnum
+name: "ClinicalAnnotation"
+url: "https://www.pharmgkb.org/page/downloadRelationshipsHelp"
```

Comment:
should be descriptionUrl-- also below; could you check to make sure that we have descriptionUrl not url for the rest of this PR?